src/Dompdf.php

```diff
@@ -875,18 +868,14 @@ public function add_info($label, $value)
     /**
      * Writes the output buffer in the log file
      *
-     * @return void
+     * @param string $logOutputFile
+     * @param float $startTime
      */
-    private function write_log()
+    private function writeLog(string $logOutputFile, float $startTime): void
     {
-        $log_output_file = $this->getOptions()->getLogOutputFile();
```

Comment:
Would it make more sense to move the log checking/creation logic into this method? It's more of a change but when we add PSR-3 support for logging the caller will just make the call and won't concern itself with the work of determining if the log exists.

Reply:
Would have to be two methods I think, because of the output buffering, which is only enabled if the log file is writable. Anyway, this was just a quick drive-by cleanup, since the method is private.